Countries Info/Managers/ApolloManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ApolloManager.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 26.11.2020.
+//
+
+import Foundation
+import Apollo
+
+class ApolloManager {
+    
+    static let shared = ApolloManager()
+    
+    private let url = URL(string: "https://countries-274616.ew.r.appspot.com/")
+    lazy var apollo = ApolloClient(url: url!)
+    
+    func fetchCountry(completion: @escaping([Country]) -> Void) {
+        
+        apollo.fetch(query: MyCountryQueryQuery()) { result in
```

Comment:
As soon as we will add new query, this part of functional can be reused, thus we can put in in separate function, for example private function within this file.